descriptors/ModuleDescriptor-template.json

```diff
@@ -1318,29 +1318,31 @@
       ]
     },
     {
-      "permissionName" : "orders-storage.claiming.process",
+      "permissionName" : "orders-storage.claiming.process.execute",
       "displayName" : "orders-storage-claiming-process",
-      "description" : "Iterate through claimed pieces and modify their status as needed"
+      "description" : "Iterate through claimed pieces and modify their status as needed",
+      "replaces": ["orders-storage.claiming.process"]
     },
     {
       "permissionName" : "orders-storage.claiming.all",
       "displayName" : "All orders-storage claiming perms",
       "description" : "All permissions for the orders-storage - claiming",
       "subPermissions" : [
-        "orders-storage.claiming.process"
+        "orders-storage.claiming.process.execute"
       ]
     },
     {
-      "permissionName" : "orders-storage.audit-outbox.process",
+      "permissionName" : "orders-storage.audit-outbox.process.execute",
       "displayName" : "orders-storage-audit-outbox-process",
-      "description" : "Fetch audit events and send them to kafka"
+      "description" : "Fetch audit events and send them to kafka",
+      "replaces": ["orders-storage.audit-outbox.process"]
     },
     {
       "permissionName" : "orders-storage.audit-outbox.all",
       "displayName" : "All orders-storage audit outbox perms",
       "description" : "All permissions for the orders-storage - audit outbox",
       "subPermissions" : [
-        "orders-storage.audit-outbox.process"
+        "orders-storage.audit-outbox.process.execute"
```

Comment:
We also need to add this change to pom.xml to avoid similar issues with naming in future.
https://github.com/folio-org/folio-module-descriptor-validator
We will do in scope of https://folio-org.atlassian.net/browse/MODORDERS-1194 planned for next sprint

![image](https://github.com/user-attachments/assets/29391d35-8369-4b73-87b7-c6adaafca630)
![image](https://github.com/user-attachments/assets/4146b90e-b611-4249-8b67-1e34224e164e)